build.gradle

```diff
@@ -22,7 +22,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-rc1'
+        classpath 'com.android.tools.build:gradle:3.0.0-rc2'
```

Comment:
I don't know if you did, but always check with @k0shk0sh before updating the gradle files.

Reply:
I can always take it out 👍

Reply:
No worries you can leave it since I will wait for him to merge. Just so you know in the future. 
I learned the hard way 🌵